src/web/Content/client/dumpling.config.json

```diff
@@ -0,0 +1,9 @@
+{
+    "dbgargs": [
```

Comment:
Why are you adding this file? As you have added it will not work as you reference your home folder that will not exist on lab machines. Did you mean to include this?

Reply:
No it got included by mistake